au4/src/trackedit/internal/trackeditactionscontroller.cpp

```diff
@@ -81,6 +87,19 @@ void TrackeditActionsController::doGlobalDelete()
     }
 }
 
+void TrackeditActionsController::doGlobalSplit()
+{
+    TrackId selectedTrackId = selectionController()->selectedTrack();
+    if (selectedTrackId != -1) {
+        dispatcher()->dispatch(TRACK_SPLIT);
```

Comment:
I think either `selected-track-split` should be sent here, or the track id should be pass to `track-split`